interfaces/apparmor/backend_test.go

```diff
@@ -941,3 +941,203 @@ func (s *backendSuite) TestSetupSnapConfineGeneratedPolicyError5(c *C) {
 	// We didn't try to reload the policy.
 	c.Assert(cmd.Calls(), HasLen, 0)
 }
+
+func (s *backendSuite) TestIsRootOverlay(c *C) {
```

Comment:
This belongs in `osutil/overlay_test.go`

Reply:
I do have these tests in overlay_test.go, but I followed the current example of TestIsHomeUsingNFS() which also tests from backend_test.go.